app/models/micropost.rb

```diff
@@ -0,0 +1,22 @@
+class Micropost < ApplicationRecord
+  MICRO_PARAMS = %i(content image).freeze
+  belongs_to :user
+  has_one_attached :image
+  scope :order_post, -> {order created_at: :desc}
+  validates :user_id, presence: true
+  validates :content, presence: true,
+    length: {maximum: Settings.mircropost.content_length}
+  validates :image, content_type: {
+    in: Settings.mircropost.img_type,
+    message: I18n.t("microposts.invalid_format_text")
+  },
+                    size: {
+                      less_than: Settings.mircropost.img_size.megabytes,
+                      message: I18n.t("microposts.invalid_size_img_text")
+                    }
+  delegate :user_name, to: :user
```

Comment:
```suggestion
  delegate :name, to: :user
```

Reply:
```suggestion
  delegate :user_name, to: :user, prefix: true
```
sau đó gọi micropost.user_name là dc